### DISCUSSION TITLE
tags is deprecated

### DISCUSSION THREAD

**Discussion by jdysya** [vuepress-theme-hope/vuepress-theme-hope#3606] [Q&A] [answered]
### Checklist

- [X] I am using **LTS** version of Node.js.
- [X] I am using the **latest v2 version** of VuePress and VuePress official plugins.
- [X] I am using the **latest v2 version** of vuepress-theme-hope and all plugins in this repo.
- [X] I followed the docs and I double checked my configuration.
- [X] I have searched [FAQ](https://theme-hope.vuejs.press/faq/) and I did not find the answer.

### Package name

vuepress-theme-hope

### Which operating system are you using?

- [X] macOS
- [ ] Windows
- [ ] Linux

### Describe the bug

I was accustomed to using the "tags" in my markdown frontmatter, however,  [the theme hope may deprecate the "tags"](https://theme-hope.vuejs.press/zh/guide/blog/category-and-tags.html#%E6%A0%87%E7%AD%BEhttps://theme-hope.vuejs.press/zh/guide/blog/category-and-tags.html#%E6%A0%87%E7%AD%BE), I wanna know why.  How can I solve this problem.

**Comment by Mister-Hope** [comment, marked as answer]
Use `tag` see https://theme-hope.vuejs.press/config/frontmatter/info.html#tag.

Maybe regexp replacement and double check?

**Reply by Mister-Hope** [reply]
Also a note for you, things like this should be posted in discussion panel (not only in my project).

Developers use issue panel to track "issues"(or problems), and a Q&A thing like this is out of scope.

We will keep supporting `tags` till stable version, so you will have enough time to migrate things.
